prow/deck/jobs/jobs.go

```diff
@@ -124,13 +138,19 @@ func (c *filteringProwJobLister) ListProwJobs(selector string) ([]prowapi.ProwJo
 
 	var filtered []prowapi.ProwJob
 	for _, item := range prowJobList.Items {
-		shouldHide := item.Spec.Hidden || c.pjHasHiddenRefs(item)
-		if shouldHide && c.showHidden {
-			filtered = append(filtered, item)
-		} else if shouldHide == c.hiddenOnly {
-			// this is a hidden job, show it if we're asked
-			// to only show hidden jobs otherwise hide it
+		if len(c.tenantIDs) != 0 && c.TenantIDMatch(item) {
```

Comment:
nit, not sure which one is better, imho it might be slightly clearer to:
```
if len(c.tenantIDs) != 0 {
  if c.TenantIDMatch(item) {
  }
}
else
```

Reply:
Done